src/pages/Article.js

```diff
@@ -92,17 +93,26 @@ const SeriesLabel = styled(SectionLabel)({
 export default connect('content')(({ content }) => (
     <article>
         <Header />
-        <Labels>
-            <SectionLabel>The NSA files</SectionLabel>
-            <SeriesLabel>Glenn Greenwald on security and liberty</SeriesLabel>
-        </Labels>
-        <Headline>{content.headline}</Headline>
-        <Standfirst
-            dangerouslySetInnerHTML={{
-                __html: content.standfirst,
-            }}
-        />
-
+        <Grid>
+            <Row>
+                <Col wide={4} leftCol={4}>
+                    <Labels>
+                        <SectionLabel>The NSA files</SectionLabel>
+                        <SeriesLabel>
+                            Glenn Greenwald on security and liberty
+                        </SeriesLabel>
+                    </Labels>
+                </Col>
+                <Col wide={12} leftCol={10}>
```

Comment:
looking at this, will we have a need for `rest` or something, when using an `inset`-like prop? something like `wide={['auto', { inset: 2 }]}`

Reply:
`inset` would be the starting column? Having an `auto` sounds like a nice API!